brainstormforce/ultimate-addons-for-gutenberg
# Fix: Blockquote - backsword hover issue on frontend.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/056b1b3d-88c0-45ee-88f8-ea603f389808?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create blockquotes with a quotation and modern layout with tweet button enable in mater.
- Add hover and normal color.
- Switch to next-release and check front end hover color not applying when master to next-release.
- Switch to above branch and regenerate the assets and check frontend again.

### Checklist:
